src/App.tsx

```diff
@@ -63,38 +66,50 @@ export const App: React.FC = () => {
         <button
           onClick={sortByName}
           type="button"
-          className={`button is-outlined is-success is-rounded ${sortType !== SortType.ALPHABET && 'is-light'}`}
+          className={classNames({
+            'button is-outlined is-success is-rounded':
+              sortType === SortType.ALPHABET,
+            'button is-outlined is-success is-rounded is-light':
+              sortType !== SortType.ALPHABET,
+          })}
```

Comment:
before you had this 
`className={`button is-outlined is-success is-rounded ${sortType !== SortType.ALPHABET && 'is-light'}`}`

from what i can see, the only optional class is 'is-light'

classnames allow you to write it cleaner

import cn from 'classnames';

cn(
  'button',
  'is-outlined',
  'is-success',
  'is-rounded',
  { 'is-light': sortType === SortType.ALPHABET },
)

what you have wrote in the previous commit:
`button is-outlined is-success is-rounded ${sortType !== SortType.ALPHABET && 'is-light'}`
will produce (`button is-outlined is-success is-rounded` + 'is-light' or +'false')

which is not correct (you have 'false' class, yikes)
what you have written in this commit is okay, but you dont need two long strings